ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml

```diff
@@ -220,25 +220,19 @@
                         <span nhs-span-type="ErrorMessage" ref="errorField" asp-validation-for="ClinicalDetails.SymptomStartDate" id="symptom-error"></span>
                         <nhs-date-input id="symptom">
                             <nhs-date-input-item>
-                                <nhs-form-group nhs-form-group-type=@SymptomGroupState>
```

Comment:
How come we're not using those anymore? Is `nhs-date-input-item` providing what we need here already?

Reply:
we are using nhs-form-group standard on all of them now